frontend/src/metabase/browse/components/BrowseModels.tsx

```diff
@@ -140,29 +139,24 @@ const ModelCell = ({ model, collectionHtmlId }: ModelCellProps) => {
     model.last_editor_common_name ?? model.creator_common_name;
   const timestamp = model.last_edited_at ?? model.created_at ?? "";
 
-  const noDescription = c(
-    "Indicates that a model has no description associated with it",
-  ).t`No description.`;
+  // const noDescription = c(
+  //   "Indicates that a model has no description associated with it",
+  // ).t`No description.`;
   return (
     <Link
       aria-labelledby={`${collectionHtmlId} ${headingId}`}
       key={model.id}
       to={Urls.model(model as unknown as Partial<Card>)}
     >
       <ModelCard>
-        <Title order={4} className="text-wrap" lh="1rem" mb=".5rem">
+        <Box mb="auto">
+          <Icon name="model" size={20} className="text-brand" />
```

Comment:
we're trying to avoid using classNames these days

```suggestion
          <Icon name="model" size={20} className={color("brand")} />
```